ansible/roles/hcv-haproxy-configure/templates/haproxy.cfg.template.j2

```diff
@@ -191,6 +191,8 @@ frontend www-http
   http-request set-var(req.ban_room) hdr(X-Room),lower,map_beg({{ haproxy_ban_prefix_map_path }})
   http-request set-var(req.ban_room) hdr(X-Group),lower,map_beg({{ haproxy_ban_prefix_map_path }})
   http-request set-var(req.ban_room) hdr(X-StickToken),lower,map_sub({{ haproxy_ban_substr_map_path }})
+  # override to unban specific groups by prefix
+  http-request set-var(req.ban_room) hdr(X-Group),lower,map_beg({{ haproxy_unban_prefix_map_path }})
```

Comment:
should this be a different variable?  won't this always set the req.ban_room and only have a ban if it matches?